internal/storage/v2/elasticsearch/tracestore/to_dbmodel.go

```diff
@@ -79,14 +79,29 @@ func resourceToDbProcess(resource pcommon.Resource) dbmodel.Process {
 		process.ServiceName = noServiceName
 		return process
 	}
+
+	var serviceName, scopeName string
 	tags := make([]dbmodel.KeyValue, 0, attrs.Len())
 	for key, attr := range attrs.All() {
-		if key == conventions.AttributeServiceName {
-			process.ServiceName = attr.AsString()
+		if key == string(conventions.ServiceNameKey) {
+			serviceName = attr.AsString()
 			continue
 		}
+		if key == string(conventions.AttributeOtelScopeName) {
+			scopeName = attr.AsString()
+		}
 		tags = append(tags, attributeToDbTag(key, attr))
 	}
+
+	switch {
+	case serviceName != "":
+		process.ServiceName = serviceName
+	case scopeName != "":
+		process.ServiceName = scopeName
```

Comment:
scope name is never a service name. Please do not change business logic in this PR.